waisntechtools/alerts/tests/test_messenger.py

```diff
@@ -0,0 +1,15 @@
+import tempfile
+
+from django.test import TestCase
+
+from alerts.messenger import Messenger, Message
```

Comment:
`Messenger` is imported but not used.

I REALLY recommend using PyCharm. PyCharm will complain about a number of syntactical issues as well as unused imports.

Also, PyCharm supports showing the diff locally in a really nice UI. It can help you review your code change before you push it up for review. I recommend this to all of the developers on my team.